.github/workflows/generate_files.yml

```diff
@@ -9,8 +9,12 @@ on:
       # to exclude it.
       - '!21.x'
 
+permissions: {}
 jobs:
   cmake:
+    permissions:
+      contents: write  #  for git push
```

Comment:
Doesn't this need read permissions as well?  This workflow doesn't run as a presubmit, so it won't fail until after you merge this PR

Reply:
`write` permission gives read as well, that's the syntax.